commercetools.Sdk/Tests/commercetools.Sdk.Linq.Tests/QueryPredicateTests.cs

```diff
@@ -750,5 +749,45 @@ public void ExpressionWithBirthDateFilters()
             string result = queryPredicateExpressionVisitor.Render(expression);
             Assert.Equal("dateOfBirth = \"1986-11-26\"", result);
         }
+
+        [Fact]
+        public void ExpressionEnumWhereInList()
+        {
+            List<OrderState> orderStates = new[] { OrderState.Open, OrderState.Complete, OrderState.Confirmed }.ToList();
+            Expression<Func<Order, bool>> expression = x => x.OrderState.In(orderStates);
+            IQueryPredicateExpressionVisitor queryPredicateExpressionVisitor = this.linqFixture.GetService<IQueryPredicateExpressionVisitor>();
+            string result = queryPredicateExpressionVisitor.Render(expression);
+            Assert.Equal("orderState in (\"Open\", \"Complete\", \"Confirmed\")", result);
+        }
+
+        [Fact]
+        public void ExpressionEnumWhereInListValueOf()
+        {
+            List<OrderState> orderStates = new[] { OrderState.Open.valueOf(), OrderState.Complete.valueOf(), OrderState.Confirmed.valueOf() }.ToList();
+            Expression<Func<Order, bool>> expression = x => x.OrderState.In(orderStates);
```

Comment:
I think we're using valueOf extension with inline variables, so i think it should be like this:
Expression<Func<Order, bool>> expression = x => x.OrderState.In(new[] { OrderState.Open, OrderState.Complete, OrderState.Confirmed }.ToList().valueOf());

Reply:
Sure thing! Updated..